data/records/jade-analysis-example.json

```diff
@@ -0,0 +1,1063 @@
+        "type": "Float_t",
+        "variable": "Yep[31]"
+      },
+      {
+        "description": "Transition parameters for CA jets at parton level, size [Nxjcp], MC trees only",
+        "type": "Float_t",
+        "variable": "Ycp[31]"
+      },
+      {
+        "description": "",
+        "type": "Char_t",
+        "variable": "Njcep[7]"
+      },
+      {
+        "description": "",
+        "type": "Char_t",
+        "variable": "Njcrp[7]"
+      },
+      {
+        "description": "Transition parameters for Durham jets at particle level, size [Nxjdh], MC trees only",
+        "type": "Float_t",
+        "variable": "Ydh[31]"
+      },
+      {
+        "description": "Transition parameters for JADE jets at particle level, size [Nxjeh], MC trees only",
+        "type": "Float_t",
+        "variable": "Yeh[31]"
+      },
+      {
+        "description": "Transition parameters for CA jets at particle level, size [Nxjch], MC trees only",
+        "type": "Float_t",
+        "variable": "Ych[31]"
+      },
+      {
+        "description": "",
+        "type": "Char_t",
+        "variable": "Njceh[7]"
+      },
+      {
+        "description": "",
+        "type": "Char_t",
+        "variable": "Njcrh[7]"
+      },
+      {
+        "description": "of partons, MC trees only",
+        "type": "Int_t",
+        "variable": "Ntrkp"
+      },
+      {
+        "description": "",
+        "type": "Char_t",
+        "variable": "Ilucp[50]"
+      },
+      {
+        "description": "of partons, size [Ntrkp], MC trees only",
+        "type": "Float_t",
+        "variable": "Ptrkp[50][4]"
+      },
+      {
+        "description": "of final hadrons, MC trees only",
+        "type": "Int_t",
+        "variable": "Ntrkh"
+      },
+      {
+        "description": "",
+        "type": "Int_t",
+        "variable": "Ntrk2"
+      },
+      {
+        "description": "of final hadrons, size [Ntrkh], MC trees only",
+        "type": "Float_t",
+        "variable": "Ptrkh[2004][4]"
+      },
+      {
+        "description": "",
+        "type": "Int_t",
+        "variable": "Iluch[2004]"
+      },
+      {
+        "description": "MC ID code of particle associated with track, size [Ntrk2], MC trees only",
+        "type": "Int_t",
+        "variable": "Iluc[501]"
+      },
+      {
+        "description": "",
+        "type": "UChar_t",
+        "variable": "Istrt[501]"
+      },
+      {
+        "description": "",
+        "type": "Char_t",
+        "variable": "Ichgh[2004]"
+      }
+    ],
+    "date_published": "2021",
+    "distribution": {
+      "formats": [
+        "C",
+        "pdf",
+        "root"
+      ],
+      "number_events": 1255308,
+      "number_files": 9,
+      "size": 73681875
+    },
+    "experiment": [
+      "JADE"
+    ],
+    "publisher": "CERN Open Data Portal",
+    "recid": "26998",
+    "title": "Examples of basic analysis techniques for the JADE detector",
+    "type": {
+      "primary": "Software",
```

Comment:
I have changed this to `Software -> Analysis` rather than `Dataset -> Derived`, based on the record title and the C file format of the example. Do you plan to release the code together with the derived dataset? Could there be more code examples for the given derived data? If so, we could  make two separate records, one for software one for data.  (This is what we usually do.)